src/py42/_internal/archive_access.py

```diff
@@ -81,12 +104,31 @@ def __init__(
         self._storage_archive_client = storage_archive_client
         self._restore_job_manager = restore_job_manager
 
-    def stream_from_backup(self, file_path):
-        metadata = self._get_file_via_walking_tree(file_path)
-        file_selection = self._build_file_selection(
-            metadata[u"path"], metadata[u"type"]
+    def stream_from_backup(self, file_paths, ignore_size_calc=False):
```

Comment:
I know this isn't new to this PR, but I encountered it while testing just now. The api expects all path separators (even for Windows paths) to be `/` so if a user enters a Windows path as they would on Windows (`C:\Users\admin\Desktop`), they'll get a File Not Found error. We should do a `.replace("\\", "/")` on the input strings.

Reply:
Good find!